philipmeneghini/InstaConnect
# Reconfigured the Backend with .NET/C# Common Etiquette

**Changes**

- Converted `TestModel` to pascal case
- Created a local MongoDB for local testing
- Removed `.env` file and moved the connection string to `appsettings.json`
- Created the `InstaConnectServices` class to be utilized by the controller
- Eliminated hard coded strings with the `ApplicationConstants` file in the Util folder

This PR would resolve issue #7 and #8

**Testing Instructions**
It is recommended to use POSTMAN to test the `GET` endpoint in the backend. First a local MongoDB database would have to be initialized. The database would have to have the following configuration:

- Database Name: InstaConnect
- Collection Name: Test
- Document Attributes: `_id` and `test`
It is also important to note that the connection string will have to be replaced with your local instance in the `appsettings.json` under "ConnectionStrings" and "Local".

Once configured locally, test the `GET` endpoint and make sure that it returns whatever is written under the `test` attribute in the local database.
